src/molecules/Tabs.tsx

```diff
@@ -45,25 +45,26 @@ const StyledLI = styled.li`
   }
 `
 
-const Tabs: React.FC = () => {
+const Tabs = withRouter(({ match }) => {
+  console.log(match )
   return (
-    <Box marginBottom="42px">
+    <Box marginBottom={3}>
       <StyldUl>
         <StyledLI>
-          <StyledLink exact to="/product/">
+          <StyledLink exact to={`${match.url}`}>
```

Comment:
If you provide a variable reference to the prop value you can omit wrapping it into a string:

```diff
- to={`${match.url}`}
+ to={match.url}
```

Wrap it in a template string when you wish to add a static part. For example,

```jsx
<Route path={`${match.url}/i-never-change`} />
```